app/api/helpers/mail.py

```diff
@@ -363,18 +363,21 @@ def send_email_to_attendees(order, purchaser_id, attachments=None):
 
 
 def send_order_cancel_email(order):
+    cancel_msg = ''
+    if order.cancel_note:
+        cancel_msg = u"<br/>Message from the organizer: {cancel_note}".format(cancel_note=order.cancel_note)
```

Comment:
No

Reply:
if the cancel_note is not none then Message from the organizer should go in the email or not? I tried this way

Reply:
Is the way of doing this wrong or something else you wanted to say?

Reply:
OK for now